utils/mil_tools/src/mil_tools/mil_tools.cpp

```diff
@@ -1,6 +1,6 @@
 #include <mil_tools/mil_tools.hpp>
-#include <mil_tools/msg_helpers.h>
-#include <mil_tools/param_helpers.h>
+#include <mil_tools/msg_helpers.hpp>
+#include <mil_tools/param_helpers.hpp>
```

Comment:
Have you tried using `find` to look for all of these? You could be sure to get them all.